sdk/src/main/java/io/outbound/sdk/OutboundPushReceiver.java

```diff
@@ -1,131 +0,0 @@
-package io.outbound.sdk;
-
-import android.app.Activity;
-import android.app.Notification;
-import android.app.NotificationManager;
-import android.content.Context;
-import android.content.Intent;
-import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.v4.content.WakefulBroadcastReceiver;
-import android.util.Log;
-
-/**
- * OutboundPushReceiver handle incoming push notifications. OutboundPushReceiver should be registered
- * as a service in your AndroidManifest.xml file.
- *
- * <p></p>
- *
- * <pre>
- * {@code
- * <receiver
- *      android:name="io.outbound.sdk.OutboundPushReceiver"
- *      android:permission="com.google.android.c2dm.permission.SEND" >
- *      <intent-filter>
- *          <action android:name="com.google.android.c2dm.intent.RECEIVE" />
- *      </intent-filter>
- * </receiver>}
- * </pre>
- *
- * <p>You may still wish to receive push notifications outside of Outbound. If you do need to handle
- * non-Outbound notifications, you can extend OutboundPushReceiver and implement {@link #onReceiveIntent(Context, Intent)}
- * method. When OutboundPushReceiver receives an intent, once it has determined that it <b>IS NOT</b>
- * an Outbound notification, it will be passed to onReceiveIntent.</p>
- */
-public class OutboundPushReceiver extends WakefulBroadcastReceiver {
```

Comment:
What's the story with this class? Was it just not being used, or is something replacing it, it's not longer needed etc?

Reply:
It's just the GCM version of OutboundMessagingService. Since the backend only interacts with FCM now (as far as I'm aware), this class never gets used